manifests/apps/jupyter/jupyter-app.yaml

```diff
@@ -202,7 +201,7 @@ spec:
 
       ii. Enter your credentials and click **Log in** (or equivalent for your identity provider).
 
-      If you see **Error 403: Forbidden**, you are not in the default user group or the default administrator group for OpenShift Data Science. Contact your administrator so that they can add you to the correct group using [Adding users for OpenShift Data Science](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/adding-users-for-openshift-data-science_useradd).
+      If you see an **Access permission needed** message, you are not in the default user group or the default administrator group for OpenShift Data Hub. Contact your administrator so that they can add you to the correct group using [Adding users for OpenShift Data Science](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_science/1/html/managing_users_and_user_resources/adding-users-for-openshift-data-science_useradd).
```

Comment:
smae here, Red Hat Openshift Data Science doc link. Text on the page talk of a dedicated-admins group that is openshift dedicated specific ...